feature-group-definitions/screen-orientation.yml

```
@@ -0,0 +1,10 @@
+name: Screen orientation
+description: The `screen.orientation` API TODO.
```

```suggestion
description: The `screen.orientation` API gets information about the orientation of the viewport, such as landscape or portrait. With this API, you can adapt an application's layout or behavior in response to changes in orientation.
```

I went back and forth a couple of times on whether to say `window.screen.orientation` or just `screen.orientation`. I'm assuming you give the latter for a reason, so I left it that way.

---

"Most customary" and using published examples to figure out what's most "most customary" seems like an _excellent_ way to approach this. Opened https://github.com/web-platform-dx/web-features/pull/917 Thank you!